cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -892,6 +909,8 @@ uint32_t VeloxShuffleWriter::calculatePartitionBufferSize(const facebook::velox:
 
   totalInputNumRows_ += numRows;
 
+  maxBatchSize_ = preAllocRowCnt == 0 ? numPartitions_ : preAllocRowCnt * numPartitions_;
```

Comment:
Can be simplified to `maxBatchSize_ = preAllocRowCnt * numPartitions_` ?

Reply:
Why adding 'numPartitions_` here? Could you add some comments here for this calculation?

Reply:
if 0 < preAllocRowCnt < 1, currently use 1 for simplicity. This calculation is not accurate.